### ISSUE TITLE
Content on atis.cloud subdomains is being indexed by Google

### ISSUE THREAD

**Issue by mikelittle** [humanmade/altis-seo#197]
Google has indexed more than 2,500 pages of content on *.altis.cloud sub domains (see  https://www.google.com/search?q=site%3Aaltis.cloud)

- The subdomains reveal the relationship between Altis/HumanMade and our clients as they all seem to have clients names in them. I understand some clients don't allow us to publicise their names as clients, this may go against that understanding.
- Some of these pages are not redirecting to their proper domains (e.g. plan-international-prod and webedia-arabia-prod)
- Some of the pages indexed are staging environments (e.g tokai-staging)
- This is bad for our clients SEO having duplicate content on these different domains.
- Some staging environment content is accessible without basic auth. e.g. https://tokai-staging.altis.cloud/uploads/2021/02/PreEnrollmentStudyGuide.pdf

Other search engines need to be checked.

Acceptance criteria:

- [ ] Requests for robots.txt made via an altis.cloud domain should return a no-follow, no-index "just go away" response. This may need to happen at the nginx level too, see next item
- [ ] Requests for uploads on xxx-staging.altis.cloud should be protected by basic-auth. This needs to happen at the nginx level.  
- [ ] Takedown requests need to be issued for all this indexed content. This will require altis.cloud being registered and verified in Google search console.

**Comment by rmccue** [comment]
> Requests for robots.txt made via an altis.cloud domain should return a no-follow, no-index "just go away" response. This may need to happen at the nginx level too, see next item
> Requests for uploads on xxx-staging.altis.cloud should be protected by basic-auth. This needs to happen at the nginx level.

Confirmed that robots.txt is accidentally [being blocked by require-login](https://github.com/humanmade/hm-require-login/blob/645caa0c96b3f47e2c1f9ff374334fc54aa7e9da/inc/namespace.php#L29-L44), which is what's causing images/etc to be indexed for the staging.

If we can fix that, I think that'll be all we need from that site; I don't think we need to go any further than that, but we could consider documenting as part of the basic auth docs that it only applies in PHP and [custom nginx conf](https://docs.altis-dxp.com/cloud/nginx-configuration/) can be used for anything else.

*Also* we should look into why any production domains aren't being redirected; @roborourke checked and it seems like it is for some customers so this might just be a config change that some have made.

**Comment by veselala** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Altis-5c378365e15075539771e4e3/issues/humanmade/altis-seo/197?planning-poker) with ZenHub @robindevitt @shadyvb @mikelittle

**Comment by veselala** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Altis-5c378365e15075539771e4e3/issues/humanmade/altis-seo/197?planning-poker) with ZenHub @roborourke

**Comment by veselala** [comment]
relates to #154

**Comment by shadyvb** [comment]
Updated acceptance criteria to reflect the result of the conversation.

**Comment by veselala** [comment]
this one is a duplicate of #154 and will close it
